Examples/Scripts/PostFreeSurferPipelineBatch-long.sh

```diff
@@ -107,158 +107,51 @@ QUEUE=""
-      --study-folder="$StudyFolder" \
-      --subject="$Subject" \
-      --surfatlasdir="$SurfaceAtlasDIR" \
-      --grayordinatesdir="$GrayordinatesSpaceDIR" \
-      --grayordinatesres="$GrayordinatesResolution" \
-      --hiresmesh="$HighResMesh" \
-      --lowresmesh="$LowResMeshes" \
-      --subcortgraylabels="$SubcorticalGrayLabels" \
-      --freesurferlabels="$FreeSurferLabels" \
-      --refmyelinmaps="$ReferenceMyelinMaps" \
-      --regname="$RegName" \
-      --longitudinal-mode="TEMPLATE" \
-      --longitudinal-template="$LongitudinalTemplate" \
-      --longitudinal-timepoint_list="${Timepoint_list[i]}" \
-      --longitudinal-timepoint="$Timepoint")
-
-  echo "Launched template job $template_job"
-  echo "Template job $template_job will wait for Stage 1 timepoint jobs: $jl" 
-  #DEBUG
-  #break;
-
-  job_list=()
-  for Timepoint in ${Timepoints[@]}; do
-  	#input variables specific for timepoint mode (if any)
-  	# ...
-	
-	#DEBUG
-	#continue
-	#process each timepoint
-	job=$($queuing_command -j $template_job "$HCPPIPEDIR"/PostFreeSurfer/PostFreeSurferPipeline.sh \
-	      --study-folder="$StudyFolder" \
-	      --subject="$Subject" \
-	      --surfatlasdir="$SurfaceAtlasDIR" \
-	      --grayordinatesdir="$GrayordinatesSpaceDIR" \
-	      --grayordinatesres="$GrayordinatesResolution" \
-	      --hiresmesh="$HighResMesh" \
-	      --lowresmesh="$LowResMeshes" \
-	      --subcortgraylabels="$SubcorticalGrayLabels" \
-	      --freesurferlabels="$FreeSurferLabels" \
-	      --refmyelinmaps="$ReferenceMyelinMaps" \
-	      --regname="$RegName" \
-	      --longitudinal-mode="TIMEPOINT_STAGE2" \
-	      --longitudinal-template="$LongitudinalTemplate" \
-	      --longitudinal-timepoint="$Timepoint")
-    job_list+=("$job")
-	#DEBUG
-	#break
-  done
-  echo "launched stage 2 timepoint jobs: ${job_list[@]}"
-  echo "Stage 2 timepoint jobs (${job_list[*]}) will wait for the template job: $template_job"
-    
-  # The following lines are used for interactive debugging to set the positional parameters: $1 $2 $3 ...
-  
-  # echo "set -- --study-folder=$StudyFolder \
-  #    --subject=$Subject \
-  #    --surfatlasdir=$SurfaceAtlasDIR \
-  #    --grayordinatesdir=$GrayordinatesSpaceDIR \
-  #    --grayordinatesres=$GrayordinatesResolutions \
-  #    --hiresmesh=$HighResMesh \
-  #    --lowresmesh=$LowResMeshes \
-  #    --subcortgraylabels=$SubcorticalGrayLabels \
-  #    --freesurferlabels=$FreeSurferLabels \
-  #    --refmyelinmaps=$ReferenceMyelinMaps \
-  #    --regname=$RegName"
-      
-  # echo ". ${EnvironmentScript}"
-done
-
+  echo Timepoints: $Timepoint_list
+
+  cmd="${HCPPIPEDIR}/PostFreeSurfer/PostFreesurferPipelineLongLauncher.sh \
+    --study-folder=\"$StudyFolder\"         \   
+    --subject=\"$Subject\"                  \
+    --template=\"$LongitudinalTemplate\"    \
+    --timepoints=\"$Timepoint_list\"        \
+    --queuing-command=\"$queuing_command\"  \
```

Comment:
Oh.  That is a bit unfortunate, arguments like this can't natively preserve arrays (and therefore whitespace).  Why does this launcher script need to be separate from the batch script?